ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -61,6 +62,7 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
     private readonly LOG_SOURCE = 'ActivityComponent';
     private shouldSaveLastStep = false;
     private isAdminEditing = false;
+    private carouselRevert:boolean = false;
```

Comment:
Just wondering what is `carousel` that should be reverted in some cases ?

Reply:
Yes, when the last form field is reached, then it starts going back